doc/en/mooncake-store.md

```diff
@@ -570,6 +570,15 @@ The HTTP metadata server can be configured using the following parameters:
 
 - **`http_metadata_server_host`** (string, default: `"0.0.0.0"`): Specifies the host address for the HTTP metadata server to bind to. Use `"0.0.0.0"` to listen on all available network interfaces, or specify a specific IP address for security purposes.
 
+#### Environment Variables
+
+- MC_STORE_CLUSTER_ID: Identify the metadata when multiple cluster share the same master, default 'mooncake'.
```

Comment:
MC_STORE_CLUSTER_ID and MC_STORE_CLUSTER_NAME should refer to the same concept. However, each mooncake cluster can support multiple models (e.g., by adding a model prefix to the key).